app/models/solr_document.rb

```diff
@@ -22,4 +22,19 @@ class SolrDocument
   def rubenstein_record?
     holdings.key?('SCL')
   end
+
```

Comment:
I think I'd prefer if these methods were in a model concern with the same module/filename as what you're overriding from Argon so that it's  little clearer that's what's happening.

Reply:
```DulArgonSkin::SmsFieldMapping```